File: URI-iniciante-Java/1020.java
import java.io.IOException;
import java.util.Scanner;
/*
 * 
*/
public class Main {
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int dias,ano,mes,dia;
		dias = sc.nextInt();
		ano=dias/365;
		mes=(dias-ano*365)/30;
		dia=(dias-(ano*365)-(mes*30));
		System.out.println(ano+" ano(s)");
		System.out.println(mes+" mes(es)");
		System.out.println(dia+" dia(s)");
		
		sc.close();
	}

}

File: Uri-iniciante/1020.cpp
#include<stdio.h>
/*	
*	-----------------------------------
*	|		Pedro Daniel Jardim       |
*	| 			  UFV				  | 
*	|		  26/02/2020			  |
*   ----------------------------------
*
*/



int main() 
{
	int dias,ano,mes,dia;
	scanf("%d",&dias);
	ano=dias/365;
	mes=(dias-ano*365)/30;
	dia=(dias-(ano*365)-(mes*30));
	printf("%d ano(s)",ano);
	printf("%d mes(es)",mes);
	printf("%d dia(s)",dia);
 

   return 0;
   
}
